addons/account/models/account_move.py

```diff
@@ -2576,12 +2576,37 @@ def button_draft(self):
     def button_cancel(self):
         self.write({'auto_post': False, 'state': 'cancel'})
 
+    def _get_mail_template(self):
+        """
+        :return: the correct mail template based on the current move type
+        """
+        return ('account.email_template_edi_credit_note' if any(move.move_type == 'out_refund' for move in self)
+                else 'account.email_template_edi_invoice')
```

Comment:
It would be nice to either prevent the user from sending both invoice + credit notes, either display a warning message on the wizard.

I would opt for the second option, but maybe it is useless; to see with POs.


________
This is a detail, but I find multiline ternary conditions much cleaner this way
```suggestion
        return (
            'account.email_template_edi_credit_note' 
            if any(move.move_type == 'out_refund' for move in self) else 
            'account.email_template_edi_invoice'
        )
```

Reply:
I made a new temp commit based on your comment and the one by tsb.
This solution is passing by the context and I'm not sure if it is a bad thing, could you give me your opinion? Thank you!